andela/phlox-ah-frontend
# # 160451462 Article Pagination

### What does this PR do?
This PR adds pagination to all article pages

### Description of Task to be completed?
Pagination for all articles

#### How should this be manually tested?
Go to the view all article URL, you should be able to go to different pages using the buttons below

###What are the relevant pivotal tracker stories?
#160451462
